twittermapping.json

```diff
@@ -106,4 +106,6 @@
   , "simonswain": "simon_swain"
   , "vaskas": "vaskas"
   , "sebarmeli": "sebarmeli"
+  , "mauricebutler": "ButlerMaurice"
```

Comment:
Ahhh... Thats what I get for editing the file on github and not actually running it up and testing it... :(